vcd/datasource_nsxt_tier0_router.go

```diff
@@ -0,0 +1,82 @@
+package vcd
+
+import (
+	"fmt"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func datasourceVcdNsxtTier0Router() *schema.Resource {
+	return &schema.Resource{
+		Read: datasourceNsxtTier0RouterRead,
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of NSX-T Tier-0 router.",
+			},
+			"nsxt_manager_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "ID of NSX-T manager.",
+			},
+			"is_assigned": &schema.Schema{
```

Comment:
`assigned` or `used` in comments explaining the complexity you are using `used`.

Reply:
Yes, but `used` is somewhat "informal" for an attribute to me. We could probably discuss `consumed` if that sounds better.
cc @dataclouder, @lvirbalas

Reply:
@dataclouder @lvirbalas  your input still would be valuable

Reply:
`assigned` is good

Reply:
`is_assigned` is fine for me in this context. BTW, it's `TypeString` ;)

Reply:
Will be `TypeBool`